resources/hooks/local/pre-push

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+GIT_CURRENT_BRANCH=$(git name-rev --name-only HEAD)
+
+# Fetch the GIT diff and format it as command input:
+DIFF=git diff --name-only origin/"$GIT_CURRENT_BRANCH"..HEAD --oneline
```

Comment:
ok with `git rev-parse HEAD` it's possible to get the latest commit in queue but I don't find a way to get the latest commit pushed.
The problem is that all the command require to have a remote reference so in any case that hook will not work without a first commit online.